mozci/task.py

```diff
@@ -339,47 +341,57 @@ def _trigger_action(self, action, payload):
         result = hooks.triggerHook(action["hookGroupId"], action["hookId"], payload)
         return result["status"]["taskId"]
 
-    def retrigger(self, push, times=3, force_retrigger=False):
-        """This function implements ability to perform retriggers on tasks.
-        The `retrigger` tag should be set to true in the task's payload, unless
-        force_retrigger parameter is set to True.
-        """
-        if not force_retrigger and self._should_retrigger() == "false":
-            logger.info(
-                "Not retriggering task '{}', task should not be retriggered".format(
-                    self.tags.get("label")
-                )
-            )
-            return None
+    def _ensure_context(self, action):
```

Comment:
This should be done as part of _get_action probably, since it's relevant to which action to select.

Reply:
Done.